tests/type-destructors/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -51,10 +50,8 @@ type Obj_Prop_x = $PropertyType<Obj, \"x\">;
 (42: Obj_Prop_x);
 
 function foo(o: Obj): $PropertyType<Obj, \"x\"> {
-  if (false)
-    return o.x;
-  else
-    return 0;
+  if (false) return o.x;
+  else return 0;
```

Comment:
There is an eslint rule, on by default in XO at least, that complains about an else clause after return in an `if` block. It would expect

```js
if (false) return o.x;
return 0
```

Is it a bridge too far to have prettier do that automatically?